invite/test/github.test.ts

```diff
@@ -48,35 +48,6 @@ describe.skip('GitHub interface', () => {
     }
   });
 
-  describe('userIsMember', () => {
```

Comment:
Why did you remove these tests?

Reply:
The send-invite API endpoint responds with a field indicating whether the invite was sent (state: pending) or the user was already a member (state: active). So, there's no point to checking if the user is a member, we can just send an invite and GitHub will ignore it if they're a member.

When the thread on limiting invites to org members or group members is resolved, this may be added for that check, but for now it can go :wastebasket: